src/lib/drivers/cri.js

```diff
@@ -140,6 +157,10 @@ class CriDriver extends Driver {
       throw new Error('connect() must be called before attempting to send a command.');
     }
 
+    if (this._commands.indexOf(command) === -1) {
+      this._commands.push(command);
```

Comment:
... It's kind of bound by the number of commands in the DevTools API :D But yeah ok.